EKF/estimator_interface.h

```diff
@@ -104,7 +104,15 @@ class EstimatorInterface
 
 	virtual void get_true_airspeed(float *tas) = 0;
 
-	virtual void get_covariances(float *covariances) = 0;
+	virtual const matrix::SquareMatrix<float, 24> covariances() const = 0;
```

Comment:
These don't need to be part of EstimatorInterface, the ekf2 frontend instantiates an Ekf instance. 
https://github.com/PX4/Firmware/blob/master/src/modules/ekf2/ekf2_main.cpp#L288

Reply:
So I just remove them?

Reply:
Yes

Reply:
Done